helpers.go

```diff
@@ -0,0 +1,33 @@
+package leaderelection
+
+import (
+	"fmt"
+	"net"
+)
+
+// GetSelfHostPort gets the global unicast address of where the application is
```

Comment:
nit: `gets the global unicast addresses`

Also, `s/where the application is running/the local execution environment/`